jarbas/core/management/commands/reimbursements.py

```diff
@@ -80,16 +77,16 @@ def serialize(self, reimbursement):
 
         return reimbursement
 
-    def bulk_create_by(self, reimbursements, size):
-        batch = list()
-        for reimbursement in reimbursements:
-            batch.append(reimbursement)
-            if len(batch) == size:
-                self.bulk_create(batch)
-                batch = list()
-        self.bulk_create(batch)
-
-    def bulk_create(self, batch):
-        Reimbursement.objects.bulk_create(batch)
-        self.count += len(batch)
-        self.print_count(Reimbursement, count=self.count)
+    def create_or_update(self, reimbursements_as_dicts):
+        for count, reimbursement in enumerate(reimbursements_as_dicts):
+            document_id = reimbursement.get('document_id')
+            if document_id:
+                Reimbursement.objects.update_or_create(
+                    document_id=document_id,
+                    defaults=reimbursement
+                )
+            self.print_count(Reimbursement, count=count + 1)
```

Comment:
I raised this question because [print_count()](https://github.com/datasciencebr/jarbas/blob/36fc5f70087e6c9aef3fb8dfe5751f6494f18e29/jarbas/core/management/commands/__init__.py#L58) takes only objects created as default. Anyway, both outputs seems relevant. What about two logs? One for created another for processed.